bitmex-stream/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+use async_stream::stream;
+use futures::SinkExt;
+use futures::Stream;
+use futures::StreamExt;
+use std::time::Duration;
+use tokio_tungstenite::tungstenite;
+use tokio_tungstenite::tungstenite::client::IntoClientRequest;
+
+pub use tokio_tungstenite::tungstenite::Error;
+
+/// Connects to the specified BitMex websocket URL and yields all messages.
+///
+/// To keep the connection alive, a websocket `Ping` will be sent every 5 seconds in case no other
+/// message was received in-between. This is according to BitMex's API documentation: https://www.bitmex.com/app/wsAPI#Heartbeats
+pub fn new(
+    request: impl IntoClientRequest + Unpin + Send + 'static,
```

Comment:
The API probably makes more sense if we hardcode the `wss://bitmex.com/...` part.